unittests/tools/test_crunch42_parser.py

```diff
@@ -0,0 +1,20 @@
+from ..dojo_test_case import DojoTestCase
+from dojo.models import Test
+from dojo.tools.crunch42.parser import Crunch42Parser
+
+
+class TestCrunch42Parser(DojoTestCase):
+
+    def test_crunch42parser_single_has_many_findings(self):
+        testfile = open("unittests/scans/crunch42/crunch42_many_findings.json")
+        parser = Crunch42Parser()
+        findings = parser.get_findings(testfile, Test())
+        testfile.close()
+        self.assertEqual(8, len(findings))
+
+    def test_crunch42parser_single_has_many_findings2(self):
+        testfile = open("unittests/scans/crunch42/crunch42_many_findings2.json")
+        parser = Crunch42Parser()
+        findings = parser.get_findings(testfile, Test())
+        testfile.close()
+        self.assertEqual(5, len(findings))
```

Comment:
Please add some field validation to you tests as well to make sure everything is being parsed and set on the finding as expected

Here is an example
https://github.com/DefectDojo/django-DefectDojo/blob/82181253cd303efd1642994327e1d862c6b247ef/unittests/tools/test_acunetix360_parser.py#L9C1-L28C131

Reply:
added in https://github.com/DefectDojo/django-DefectDojo/pull/9689/commits/003b713b7d39b489386728d32743c36f624488b7